### PR TITLE
Add link to prompting guide within the dashboard UI #812

### PR DESCRIPTION
**Pull request by sir-braz**
Adding easier link to the prompting guide from within the dashboard UI. Please check that the position of the button is correct.

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "eslint-plugin-simple-import-sort": "^8.0.0",
     "focus-visible": "^5.2.0",
     "framer-motion": "^6.5.1",
+    "i18next": "^22.4.9",
```
Comment (andreaskoepf) [inline website/package.json:50]: Does this PR need i18next?

File: website/src/components/Layout.tsx [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 import { Box, Grid } from "@chakra-ui/react";
 import type { NextPage } from "next";
-import { FiBarChart2, FiLayout, FiMessageSquare, FiUsers } from "react-icons/fi";
+import { FiAlertTriangle, FiBarChart2, FiLayout, FiMessageSquare, FiUsers } from "react-icons/fi";
```
Comment (andreaskoepf) [inline website/src/components/Layout.tsx:5]: `FiAlertTriangle` seems not to be referenced.

File: website/src/components/Layout.tsx [hunk 3]
```diff
@@ -52,6 +54,12 @@ export const getDashboardLayout = (page: React.ReactElement) => (
           desc: "User Leaderboard",
           icon: FiBarChart2,
         },
+        {
+          label: "Guide",
+          pathname: "/guide",
```
Comment (andreaskoepf) [inline website/src/components/Layout.tsx:59]: There isn't anything at `/guide` is it?

**Review by andreaskoepf — COMMENTED** [review]
Is this still in draft stage?

File: website/src/components/Layout.tsx (hunk 3 above)
Comment (sir-braz) [reply website/src/components/Layout.tsx:59]: There is nothing in /guide, it is under development

File: website/package.json (hunk 1 above)
Comment (sir-braz) [reply website/package.json:50]: It is not necessary !

**Comment by fozziethebeat** [comment]
I've resolved the underlying issue with a simpler solution

**Pull request closed without merging**